packages/bot-web-ui/src/pages/tutorials/quick-strategy-content/quick-strategy-guides-details.tsx

```diff
@@ -25,6 +25,13 @@ const QuickStrategyGuidesDetail = observer(
         const { is_mobile } = ui;
         const text_size = is_mobile ? 'xs' : 's';
 
+        const qs_guide = document.querySelector('.tutorials-mobile__qs-guide');
+        const scrollToTop = () => {
```

Comment:
```suggestion
        const scrollToTop = () => {
                const qs_guide = document.querySelector('.tutorials-mobile__qs-guide');
```
Small nit: 
1. putting the `qs_guide` declaration outside in the component scope would mean it will only be updated when the component re-renders
2. putting it within the `scrollToTop` scope would mean that it will be updated on each function call

Doing it 1. would be problematic if it changes frequently. Not a problem if it rarely changes.
Doing it 2 would ensure that u will always get the most updated element not tied to the component's lifecycle